mobile-browser-based-version/src/helpers/training/training_manager.js

```diff
@@ -144,79 +148,215 @@ export class TrainingManager {
   async _trainLocally() {
     const info = this.task.trainingInformation;
 
-    await this.model.fit(this.data, this.labels, {
-      initialEpoch: this.model.getUserDefinedMetadata().epoch,
-      epochs: info.epochs ?? MANY_EPOCHS,
-      batchSize: info.batchSize,
-      validationSplit: info.validationSplit,
-      shuffle: true,
-      callbacks: {
-        onEpochEnd: async (epoch, logs) => {
-          this.trainingInformant.updateGraph(
-            epoch + 1,
-            (logs.val_acc * 100).toFixed(2),
-            (logs.acc * 100).toFixed(2)
-          );
-          console.log(
-            `EPOCH (${epoch + 1}):
+    if (info.batchwise) {
```

Comment:
info.batchwise, what does this represent? It's not clear to me at a first glance (might be worth making the name longer and explicit for legibility)